src/app/filter/transfers/transfers.component.ts

```diff
@@ -0,0 +1,56 @@
+import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
+import {FormBuilder, FormGroup} from "@angular/forms";
+import {MatCheckboxChange} from "@angular/material/checkbox";
+
+@Component({
+  selector: 'app-transfers',
+  templateUrl: './transfers.component.html',
+  styleUrls: ['./transfers.component.scss']
+})
+export class TransfersComponent implements OnInit {
+
+    @Input() segments: any;
+    @Output() checked: EventEmitter<any> = new EventEmitter();
+    transfers: FormGroup;
+    isChecked: boolean | undefined;
```

Comment:
Ты можешь использовать знак вопроса для переменных которые могут быть не проинициализированные 

```suggestion
    isChecked?: boolean;
```